msgpack.d.ts

```diff
@@ -1,16 +1,28 @@
 /**
  * Serializes a value to a MessagePack byte array.
- * @param data The value to serialize. This can be a scalar, array or object.
- * @param multiple Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.
+ * @param data An object that defined additional options.
+ * @param options Indicates whether multiple values in data are concatenated to multiple MessagePack arrays.
  */
-export function serialize(data: any, multiple: boolean = false): Uint8Array;
+export function serialize(
+    data: any,
+    options: {

```

Comment:
Does that mean that the `options` parameter must be passed? The implementation doesn't require it, it can be left out to use the defaults. Same for `deserialize`.

Reply:
Whoops. Yes, my mistake. I'll get that fixed.